Tests/Hotels/Signup.feature

```diff
@@ -4,7 +4,7 @@ Feature: Sign Up
   @TC-22
   Scenario Outline: Verify error message for invalid data in SignUp form
     When I Lunch hotels.com homepage
```

Comment:
Lunching the page, where you start your test is a given/precondition step.
When => Given